feat: Allow @api_view wrapped functions as well.

bikemule opened the pull request:
Thank you for making this! I was looking for something like this and was only able to find the older project: https://bitbucket.org/hub9/django-hybrid-router/src/master/ 

It allows both `APIView` and `@api_view` in addition to `ViewSet`s, but I had some issues (mainly not being able to prefix/nest things).

So I came back to your project and made the changes I needed to make it work.

I ran the tests with `poetry run pytest` and they all passed.

Let me know your thoughts!

enzofrnt commented:
Hi, 

thanks for your contribution.
Please write test that verify the behaviors that you expect. Also please make the CI run.

Thanks you !

enzofrnt commented:
@bikemule any news ?

bikemule commented:
Hi,

Sorry I didn't see the notifications before.

Last time I didn't run pre-commit and when I did I got some errors that I fixed.

I added a test ,`test_register_only_api_views` and I modified `test_add_views_and_viewsets` to also add an `api_view`.

I ran pre-commit this time, everything passed.

I ran CI on Github and the only issue was a codecov rate limit error.

Let me know if I should change anything.

enzofrnt commented:
Hi, @bikemule thanks you. I should make a fix on the CI, and then please rebase your branch thanks

enzofrnt commented:
The modification should avoid the GitHub action issue
